src/plugins/theme/borderWidth.js

```diff
@@ -1,17 +1,11 @@
 /**
- * Border width
- * Note: These are in px not rem units
+ * Note: these are in px not rem units
  */
 module.exports = function () {
   return {
-    DEFAULT: '1px',
-    0: '0',
-    2: '2px',
     3: '3px',
-    4: '4px',
     5: '5px',
     6: '6px',
     7: '7px',
```

Comment:
This is an example of what I meant by we were already extending without explicitly using the "extend" format. TW by default provides 1, 0, 2, 4, and 8px. We copied these existing values previously and added the extra 3, 5, 6 and 7px